### PR TITLE
Fixed errors caused by adding set -e. #617

### PR DESCRIPTION
**Pull request by btootomoyuk**
Please review #617 .

### CODE REVIEW

**Review by btiwahorisu — APPROVED** [review]

File: create-maven-archetype.sh [hunk 1]
```diff
@@ -72,7 +72,9 @@ if [ -d projectName-domain/src/main/resources/xxxxxx ];then
 fi
 
 sed -i -e "/REMOVE THIS LINE IF YOU USE $ORM/d" `grep -rIl $ORM projectName-* | grep -v '.svn'`
-sed -i -e "s/REMOVE THIS COMMENT IF YOU USE $ORM//g" `grep -rIl $ORM projectName-* | grep -v '.svn'`
+if [ $CONFIG == "JavaConfig" ]; then
+  sed -i -e "s/REMOVE THIS COMMENT IF YOU USE $ORM//g" `grep -rIl $ORM projectName-* | grep -v '.svn'`
+fi
```
Comment (btkatoufm) [inline create-maven-archetype.sh:77]: - [ ] Instead of branching in JavaConfig, why not just store the list of files in a variable?

Comment (btootomoyuk) [reply create-maven-archetype.sh]: https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/pull/622/commits/1fd96b008e2f37636c1c4534a25183d4879707d4
Modified to store the list of files in a variable.

Comment (btkatoufm) [reply create-maven-archetype.sh]: Just to leave this as a comment, the script assumes that the result of grep with $ORM exists. Therefore, if the grep result does not exist, the set -e will stop the process, which is the correct behavior, so we assume that there is no problem with this response.

**Comment by btkatoufm** [comment]
relates to [TSF4J5-2820](https://terasolunaorg.atlassian.net/browse/TSF4J5-2820).

**Pull request merged**